grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunction.java

```diff
@@ -57,7 +57,7 @@ static GrpcExceptionHandlerFunction of() {
      * internally in Armeria, no {@link Status} created, so {@code null} will be specified.
      */
     @Nullable
-    Status apply(RequestContext ctx, @Nullable Status status, Throwable cause, Metadata metadata);
+    Status apply(RequestContext ctx, Status status, Throwable cause, Metadata metadata);
```

Comment:
The ' Throwable ' parameter can be removed because the `Status` is not nullable and always has a cause.
Let me know if it's acceptable to make the breaking change here again.

Reply:
I prefer the current signature because `Status.getCause()` is marked as `@Nullable`.
IDE may complain if `Status.getCause()` is accessed without null checking.